src/renderer/src/components/ContextFiles/ContextFiles.tsx

```diff
@@ -352,6 +353,17 @@ export const ContextFiles = ({ baseDir, allFiles, showFileDialog }: Props) => {
                   <span
                     className={`select-none text-2xs overflow-hidden ${item.isFolder ? 'context-dimmed' : 'text-text-primary font-semibold'}`}
                     {...(item.isFolder ? { onClick: context.arrowProps.onClick } : {})}
+                    data-tooltip-id="context-files-tooltip"
```

Comment:
Let's also add `data-tooltip-delay-show={800}` so there is a higher delay for this tooltip.